classes/module/Module.php

```diff
@@ -866,6 +889,21 @@ public function disableDevice($device)
         return true;
     }
 
+    public function isDeviceEnabled(int $device): bool
+    {
+        $query = new DbQuery();
+        $query->select('enable_device');
+        $query->from('module_shop', 'ms');
+        $query->where('ms.id_module = ' . (int) $this->id);
+
+        $row = Db::getInstance()->getRow($query);
+        if (!is_array($row)) {
+            return false;
+        }
+
+        return (bool) ($row['enable_device'] & $device);
+    }
```

Comment:
What's the purpose of this function? It looks like you're only using it for testing purpose?

Reply:
Affirmative

Reply:
Isn't that a bit weird to have a new function only for testing purpose? Can't you make a similar function in the test class instead?

Reply:
I don't know. I think this `getter` is useful for using.

Reply:
What do you think @PrestaShop/prestashop-core-developers ?